src/Simplex/Messaging/Notifications/Server/Store/Postgres.hs

```diff
@@ -267,8 +271,8 @@ foldNtfSubscriptions st srv fetchCount state action =
         WHERE p.smp_host = ? AND p.smp_port = ? AND p.smp_keyhash = ?
           AND s.status IN ?
       |]
-    params = srvToRow srv :. Only (In [NSNew, NSPending, NSActive, NSInactive])
-    opts = DB.defaultFoldOptions {DB.fetchQuantity = DB.Fixed fetchCount}
+    orderLimit = " ORDER BY s.subscription_id LIMIT ?"
```

Comment:
Add explicit ASC or DESC, seems to be ASC according to logic?..

Reply:
doesn't matter, but why?